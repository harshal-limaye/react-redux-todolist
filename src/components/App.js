import React from 'react';
import Header from './Header';
import CreateTodo from './CreateTodo';
import TodoList from './TodoList';

export default class App extends React.Component {
    render() {
        return (
            <div className="container">
                <div className="todo-app">
                    <Header />
                    <CreateTodo />
                    <TodoList />
                </div>
            </div>
        );
    }
}