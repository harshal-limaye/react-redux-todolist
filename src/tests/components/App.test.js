import React from 'react';
import { shallow } from 'enzyme';
import App from '../../components/App';

describe('testing app component', () => {
    test('should render component correctly', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
});