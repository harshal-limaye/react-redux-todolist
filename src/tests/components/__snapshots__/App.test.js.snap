// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing app component should render component correctly 1`] = `
<div
  className="container"
>
  <div
    className="todo-app"
  >
    <Header />
    <Connect(CreateTodo) />
    <Connect(ToDoList) />
  </div>
</div>
`;
